src/batch-client/index.ts

```diff
@@ -306,23 +306,34 @@ export class ZimbraBatchClient {
 		const { attributes, ...rest } = data;
 		const contactAttrs = <Object[]>[];
 
-		forEach(attributes, (val, key) =>
-			contactAttrs.push({
-				name: key,
-				[key === 'image' ? 'aid' : 'content']: val
-			})
+		forEach(
```

Comment:
Can we add this entire flow to a function may be? it is repeating in modifyContact as well.

Reply:
Added